### PR TITLE
Place layout of the placeholder for the second part of the project

### PR DESCRIPTION
**Pull request by Zibilyonik**
-Adjust the appearance of the background
-Create a card layout placeholder for the next step of the project.
-Create Todo Class and add function to make the js file more readable.
-Edit the README file to properly explain the project and how to use the Todo List tool.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,9 +1,17 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
   <head>
     <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
```
Comment (carlos-ssh) [inline src/index.html:6]: - Kindly enable your linters into your project. 
- Make sure that there are no linter errors.

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi  @Zibilyonik , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.html (hunk 1 above)
Comment (Zibilyonik) [reply src/index.html:6]: Hello, I've already done that right now, but I'm getting an error from Lighthouse. Can you take a look again and give me advice on how I can solve this?
Thank you for your time.

Comment (carlos-ssh) [reply src/index.html:6]: Hi @Zibilyonik, how are you? check your linter files if you set up that files correctly.

**Pull request merged**